src/device/wg_controller.rs

```diff
@@ -2095,6 +2113,7 @@ mod tests {
             pub_key,
             Some(Duration::from_secs(DEFAULT_PEER_UPGRADE_WINDOW)),
         )]);
+        f.then_cpc_unpaused(pub_key);
```

Comment:
A bit unfortunate that cannot easily test `cpc_paused()` as consolidation would need to be called multiple times for that to happen.